Library/Formula/lowprofile.rb

```diff
@@ -0,0 +1,50 @@
+require "language/go"
+
+class Lowprofile < Formula
+  desc "Command-line utility for AWS profile management."
+  homepage "https://github.com/DualSpark/lowprofile"
+  url "https://github.com/DualSpark/lowprofile/archive/v0.1.tar.gz"
+  sha256 "056834b644ea01b2244babb13ed462077d1df6361524f124df0c3f31639db938"
+
+  depends_on "go" => :build
+
+  go_resource "github.com/DualSpark/lowprofile" do
+    url "https://github.com/DualSpark/lowprofile.git", :revision => "v0.1"
+  end
+
+  def install
+    ENV["GOPATH"] = buildpath
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    # Build and install lowprofile
+    system "go", "build", "-v", "-o", "./bin/lowprofile-#{version}", "main.go"
```

Comment:
Use `"#{bin}/lowprofile-#{version}"` instead of `"./bin/lowprofile-#{version}"` and remove the `bin.install` line below.

Also why include the version here? Shouldn’t that be installed as `lowprofile` instead of `lowprofile-0.1`?